src/block/block_compact.c

```diff
@@ -18,8 +18,12 @@ int
 __wt_block_compact_start(WT_SESSION_IMPL *session, WT_BLOCK *block)
 {
 
-    if (block->compact_session_id != WT_SESSION_ID_INVALID)
+    if (block->compact_session_id != WT_SESSION_ID_INVALID) {
+        __wt_verbose_info(session, WT_VERB_COMPACT,
+          "Compaction already happening on data handle %s by session %p, returning EBUSY.",
```

Comment:
Is the `session` address worth it?

Reply:
Session ID would be more useful than the address if we want to debug and find out which session is running the compact.

Reply:
Sounds good, should I use `WT_RET_MSG` instead as well?

Reply:
Added session ID in 8a390281a94310fd6c5a24cd6cb7a3843713e54c.

Reply:
I feel like returning the error message with `WT_RET_MSG` is more useful than a verbose message. Do you think of any scenario where these error messages can be too many? If not, `WT_RET_MSG` is a good option.

Reply:
I agree `WT_RET_MSG` makes sense here.

Reply:
This should not happen often unless the application keeps triggering compaction on the same table from different sessions.
Update with `WT_RET_MSG` in a0ba303c929c17d193247b3c70be1948e71f8626.